ramls/definedByObject.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "title" : "Okapi Permission Object",
```

Comment:
Is this name more self-explanatory?
ramls/permissionOwner.json
"title" : "Owner (Module or User) that defines an Okapi Permission"

Reply:
we decided to remove this definedBy property altogether.  The `moduleName` & `moduleVersion` properties will live directly under permission.